controllers/highlight.js

```diff
@@ -48,5 +49,15 @@ export default {
       return res.json({ message: 'You have no highlights yet!' });
     }
     return res.json({ highlights });
+  },
+
+  async removeHighlight(req, res) {
+    const { params: { id }, decoded } = req;
+    const highlight = await Highlight
```

Comment:
How do we get to the highlight without an article?

In my opinion, the highlight should come attached with the slug as part of the params so that we can trace the article it belongs to. 

In essence, there may have to be some modifications on the PR we merged earlier such that a highlight related endpoint has the slug as well, and probably the highlight table should have the slug also. Maybe the slug is in the end more important than the `articleId`. Just the same way we related comments to articles.